sinch/sinch-sdk-java
# Feature: verification API & webhooks

This PR is just the sum of the ones related to Verification API
So request to merge to main the previously approved:
- Add `Verification` API support
- Add `WebHooks` helpers to serialize/deserialize
- Add helper to check `Authentication from webhooks`
